scripts/load_templates.sh

```diff
@@ -4,7 +4,7 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 cd "${DIR}"/.. || exit
 
-DESIGN_SYSTEM_VERSION="6.1.0"
+DESIGN_SYSTEM_VERSION="8.0.1"
```

Comment:
Not really related but was there a reason we did not use an env var for this?. Means we would be able to upgrade the non-breaking changes without app changes.

Reply:
Because we would always want this to be fixed and not change without tests